### PR TITLE
Add basic sketch for cpgqls-client-python

### PR DESCRIPTION
**Pull request by ursachec**

### CODE REVIEW

File: cpgqls-client-python/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## cpgqls-client
+
+`cpgqls-client` is a simple Python library for communicating with an instace of
```
Comment (fabsx00) [inline cpgqls-client-python/README.md:3]: instace => instance

File: cpgqls-client-python/LICENSE [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+========================================================================
+MIT Licenses
+========================================================================
+
+The Apache TinkerPop project bundles the following components under the MIT License:
```
Comment (fabsx00) [inline cpgqls-client-python/LICENSE:208]: I think we're not using tinkerpop in this lib, so, if we place a separate license in the lib folder, then we should omit references to it.

Comment (ursachec) [reply cpgqls-client-python/LICENSE:208]: My bad, updated

**Comment by ursachec** [comment]
Discussed IRL, moving the library to a separate repo, will close the PR and point to it as soon as the task is done

**Comment by ursachec** [comment]
Moved to different repository https://github.com/ShiftLeftSecurity/cpgqls-client-python

**Pull request closed without merging**